### DISCUSSION TITLE
Is there a .pintarc ?

### DISCUSSION THREAD

**Discussion by gsilos** [PintaProject/Pinta#1023] [Q&A]
Is there a $HOME/.pintarc where I can set defaults for Brush width for example? Every time I open pinta, I need to set Brush width from 2 to 8. I do this a lot of time a day. I guess it will save time and clicks when using pinta.

Note: If it is not .pintarc, is there a solution to persist Brush width ?

**Comment by cameronwhite** [comment]
Which version of Pinta are you using? The tools should save their last values to the preferences file ( typically `~/.config/Pinta/settings.xml` ) , which I think was added in Pinta 2.0
